Lib/test/test_struct.py

```diff
@@ -578,6 +578,26 @@ def test__sizeof__(self):
         self.check_sizeof('0s', 1)
         self.check_sizeof('0c', 0)
 
+    def test_boundary_error_message(self):
+        byte_list = bytearray(1)
+        with self.assertRaisesRegex(
+                struct.error,
+                'pack_into requires a buffer of at least 6 '
```

Comment:
This is just a style nit, but I find this style is a bit ugly. I prefer something like this:

```py
regex = (
    r'multi'
    r'line'
    r'regex'
)

with self.assertRaisesRegex(struct.error, regex):
    ...
```